scripts/dl_cleanup.py

```diff
@@ -90,6 +98,7 @@ def parseVer_GIT(match, filepath):
 
 versionRegex = (
 	(re.compile(r"(.+)[-_](\d+)\.(\d+)\.(\d+)\.(\d+)"), parseVer_1234),	# xxx-1.2.3.4
+	(re.compile(r"(.+)[-_](\d\d\d\d)-?(\d\d)-?(\d\d)-"), parseVer_ymd),	# xxx-YYYY-MM-DD-GIT_SHASUM
```

Comment:
Shouldn't this reference the new function parseVer_ymd_GIT_SHASUM?

Reply:
yes it's a typo